functional-tests/tests/test_anonymizer.py

```diff
@@ -67,6 +68,18 @@ def test_anonymize_api_fails_on_invalid_value_of_text():
     assert response_status == 422
 
 
+@pytest.mark.api
+def test_anonymize_with_redact_and_replace():
```

Comment:
our naming convention can go here as well

Reply:
Lets ask in the thread if it applies to both.

Reply:
don't see why not, conventions are good, will go over the rest of the tests in a future refactor I want to do in the functional tests

Reply:
I am not saying it's not a good idea... just not sure a unit test convention matches functional tests.
I raised it in the chat.